roles/download/defaults/main.yml

```diff
@@ -80,6 +81,9 @@ kubedns_image_repo: "gcr.io/google_containers/kubedns-amd64"
 kubedns_image_tag: "{{ kubedns_version }}"
 test_image_repo: busybox
 test_image_tag: latest
+vault_deployment_type: "docker"
```

Comment:
Please remove this. We have legacy ways to deploy some components, but it doesn't look like you're introducing a package/curl based method to deploy vault, so this is just an extra parameter floating around that doesn't get evaluated for anything